course/tests.py

```diff
@@ -1,3 +1,29 @@
+from django.core.management import CommandError, find_commands
 from django.test import TestCase
+from django.core import management
+from io import StringIO
+import sys
 
-# Create your tests here.
+
+class CommandTests(TestCase):
+    databases = '__all__'
+    fixtures = ["dumps/courseData.json"]
+
+    def test_command(self):
+        sys.stdout = out = StringIO()
+        management.call_command('exportcourse', 1, stdout=out, verbosity=3)
+        self.assertIn("Exporting course Spanish for English speakers course", out.getvalue())
+
+    def test_calling_a_command_without_argument(self):
```

Comment:
I don't understand what this is testing? Isn't this logic included in django? We only need to test our own code, not django default functionality

Reply:
Which line number are you talking about? 15 or 17?

Reply:
In line 15, just checking if this text is in console output or not.

Reply:
In line 17, we are checking if we pass the argument or not.